now.json

```diff
@@ -4,7 +4,7 @@
     "alias": [
         "magento-venia-develop"
     ],
-    "scope": "magento-research",
+    "scope": "magento",
```

Comment:
This is the [now.sh scope](https://zeit.co/docs/v2/advanced/configuration#scope), not the github scope. Does it exist at now.sh and if so, who has access and how can we migrate our current accounts/profiles to it? If a magento team at now.sh doesn't exist it may be as simple as updating our team url/name:

[![Image from Gyazo](https://i.gyazo.com/7c4ab7e4602b860dc5fba2d7954b1874.png)](https://gyazo.com/7c4ab7e4602b860dc5fba2d7954b1874)

Reply:
My point is...maybe we don't have to do this through the migration. Perhaps it'll just keep working :D